### PR TITLE
Patch/impacted entities typo

### PR DESCRIPTION
**Pull request by jetzlstorfer**
Fix typo with `ImpactedEntity` that was accidentially referring to `ImpactedEntities`.
Fix typo with `FileChangesUmbrellaChart`

### CODE REVIEW

File: pkg/events/keptnEvents.go [hunk 1]
```diff
@@ -222,7 +222,7 @@ type ProblemEventData struct {
 	// PID is a unique system identifier of the reported problem.
 	PID string `json:"PID"`
 	// ImpcatedEntity is an identifier of the impacted entity
-	ImpactedEntity string `json:"ImpactedEntities,omitempty"`
+	ImpactedEntity string `json:"ImpactedEntity,omitempty"`
```
Comment (christian-kreuzberger-dtx) [inline pkg/events/keptnEvents.go]: I just checked the problem events we are working with, and Dynatrace sends both, the `ImpactedEntities` and the `ImpactedEntity` field. We only forward the `ImpactedEntity` field though. 
I believe the change here doesn't break anything, so I'm okay with it, but I'll leave it to @johannes-b and @agrimmer to decide whether we want this or not.

This is a the `.problem` event sent by dynatrace:
```json
 {
      "contenttype": "application/json",
      "data": {
        "ImpactedEntities": [
          {
            "entity": "SERVICE-B4A42841E92B2772",
            "name": "carts.production.primary",
            "type": "SERVICE"
          }
        ],
        "ImpactedEntity": "Response time degradation on Web service carts.production.primary",
        "PID": "1689062257091243766",
        "ProblemDetails": {
          "affectedCounts": {
            "APPLICATION": 0,
            "ENVIRONMENT": 0,
            "INFRASTRUCTURE": 0,
            "SERVICE": 0
          },
          "commentCount": 0,
          "displayName": "766",
          "endTime": 1579074900000,
          "hasRootCause": true,
          "id": "1689062257091243766_1579072620000V2",
          "impactLevel": "SERVICE",
          "rankedEvents": [
            {
              "affectedRequestsPerMinute": 141,
              "endTime": 1579075200000,
              "entityId": "SERVICE-B4A42841E92B2772",
              "entityName": "carts.production.primary",
              "eventType": "SERVICE_RESPONSE_TIME_DEGRADED",
              "impactLevel": "SERVICE",
              "isRootCause": true,
              "percentile": "50th",
              "referenceResponseTime50thPercentile": 0,
              "referenceResponseTime90thPercentile": 0,
              "service": "carts.production.primary",
              "serviceMethodGroup": "Default requests",
              "severities": [
                {
                  "context": "RESPONSE_TIME_50TH_PERCENTILE",
                  "unit": "MicroSecond (µs)",
                  "value": 881916
                }
              ],
              "severityLevel": "PERFORMANCE",
              "startTime": 1579072620000,
              "status": "CLOSED",
              "userDefined50thPercentileThreshold": 100000,
              "userDefined90thPercentileThreshold": 1000000
            }
          ],
          "rankedImpacts": [
            {
              "entityId": "SERVICE-B4A42841E92B2772",
              "entityName": "carts.production.primary",
              "eventType": "SERVICE_RESPONSE_TIME_DEGRADED",
              "impactLevel": "SERVICE",
              "severityLevel": "PERFORMANCE"
            }
          ],
          "recoveredCounts": {
            "APPLICATION": 0,
            "ENVIRONMENT": 0,
            "INFRASTRUCTURE": 0,
            "SERVICE": 1
          },
          "severityLevel": "PERFORMANCE",
          "startTime": 1579072620000,
          "status": "CLOSED",
          "tagsOfAffectedEntities": [
            {
              "context": "CONTEXTLESS",
              "key": "keptn_service",
              "value": "carts"
            },
            {
              "context": "CONTEXTLESS",
              "key": "keptn_stage",
              "value": "production"
            },
            {
              "context": "CONTEXTLESS",
              "key": "keptn_deployment",
              "value": "primary"
            },
            {
              "context": "CONTEXTLESS",
              "key": "keptn_project",
              "value": "cksockshop3"
            }
          ]
        },
        "ProblemID": "766",
        "ProblemTitle": "Response time degradation",
        "State": "RESOLVED",
        "Tags": "keptn_service:carts, keptn_deployment:primary, keptn_stage:production, keptn_project:cksockshop3",
        "eventContext": {
          "keptnContext": "b6124c67-0b86-4c34-9029-7ce01f3c16d5",
          "token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE2MTAxNzkyODl9.kWHZIrcmd7i0GFMIQbRj5Dt-w5CyWumaOENda6STkqg"
        }
      },
      "id": "667ab84a-d45e-4e43-a99d-879bab976ff9",
      "source": "https://github.com/keptn/keptn/api",
      "specversion": "0.2",
      "time": "2020-01-15T08:01:29.897Z",
      "type": "sh.keptn.events.problem",
      "shkeptncontext": "b6124c67-0b86-4c34-9029-7ce01f3c16d5"
    }
```

File: pkg/events/keptnEvents.go [hunk 2]
```diff
@@ -93,9 +93,9 @@ type ConfigurationChangeEventData struct {
 	// FileChangesGeneratedChart provides new content for the generated chart.
 	// The key value pairs represent the URI within the chart (i.e. the key) and the new content (i.e. the value).
 	FileChangesGeneratedChart map[string]string `json:"fileChangesGeneratedChart,omitempty"`
-	// FileChangesUmbrellaChart provides new content for the umbrealla chart.
+	// FileChangesUmbrellaChart provides new content for the umbrella chart.
 	// The key value pairs represent the URI within the chart (i.e. the key) and the new content (i.e. the value).
-	FileChangesUmbrellaChart map[string]string `json:"fileChangesUmbreallaChart,omitempty"`
+	FileChangesUmbrellaChart map[string]string `json:"fileChangesUmbrellaChart,omitempty"`
```
Comment (christian-kreuzberger-dtx) [inline pkg/events/keptnEvents.go]: Thanks for fixing this typo. @agrimmer does this break anything (thinking about external services)?

File: pkg/events/keptnEvents.go (hunk 1 above)
Comment (johannes-b) [reply pkg/events/keptnEvents.go]: The example in the keptn/spec also says `ImpactedEntity` and I double-checked the current implementation of the serivcenow-service where also the `ImpactedEntity` is used.  Change accepted.

File: pkg/events/keptnEvents.go (hunk 2 above)
Comment (agrimmer) [reply pkg/events/keptnEvents.go]: Change accepted.

**Pull request merged**